packages/frontend/src/components/accounts/create/verify_account/VerifyAccountWrapper.js

```diff
@@ -38,10 +39,17 @@ export function VerifyAccountWrapper() {
 
     useEffect(() => {
         const checkIfMoonPayIsAvailable = async () => {
-            const moonpayAvailable = await isMoonpayAvailable();
-            if (moonpayAvailable) {
-                setShowFundWithCreditCardOption(true);
-            }
+            await Mixpanel.withTracking("CA Check Moonpay available",
+                async () => {
+                    const moonpayAvailable = await isMoonpayAvailable();
```

Comment:
Prior to this change, if `isMoonpayAvailable()` were to throw an error, it would bubble up somewhere.
After this change, it will be swallowed by the error handle on L49-L51 and won't bubble up

Is that an intentional change in behaviour here?

Reply:
Changed it to `throw e` :)